ingest/expression_files/mtx.py

```diff
@@ -142,6 +194,54 @@ def get_features(feature_row: str):
             gene_name = feature_data[1]
         return gene_id, gene_name
 
+    @staticmethod
+    def sort_mtx(file_path) -> str:
+        """
+        Sorts MTX file by gene. File header, dimensions, and comments are not included in sort.
+
+         Returns
+         ----------
+            new_file_path (str) : Full path of newly sorted MTX file
+        """
+        file_name = ntpath.split(file_path)[1]
+        new_file_name = f"{file_name}_sorted_MTX.mtx"
+
+        with open(new_file_name, "w+") as f:
+            GeneExpression.dev_logger.info("Starting to sort")
+            start_time = datetime.datetime.now()
+            # Line to start sorting at
+            start_idx: int = MTXIngestor.get_data_start_line_number(file_path)
+            # Grab gene expression data which starts at 'n', or start_idx, lines from top of file (-n +{start_idx}).
+            # Transform() is expecting the file handle to be at the first line of data which is why sorting starts at
+            # line number 'n', as defined by start_idx
+            p1 = subprocess.Popen(
+                ["tail", "-n", f"+{start_idx}", f"{file_path}"], stdout=subprocess.PIPE
```

Comment:
For readability, omit f-string when not concatenating:

```suggestion
                ["tail", "-n", f"+{start_idx}", file_path], stdout=subprocess.PIPE
```